src/graphics/start.h

```diff
@@ -0,0 +1,176 @@
+#ifndef START_H
+#define START_H
+
+#include "../../include/raylib.h"
+
+#include "tmp_structs.h"
+#include "constants.h"
+/**
+ * The `initStartMenu` method creates the initial data for the Menu struct
```

Comment:
Don't use the backtick for variable names (" \` "). This is something that is used in markdown, not in general formats. If you think that there is something that the reader might confuse between a word and a variable, then use the `$` to identify the variable.

Example of this comment:
```
Initializes the $Menu struct with default values.
@param menu is a pointer to the Menu struct to be initialized.
...
```

Also, this function **DOES NOT** allocate memory, your code is good, but your comments are inaccurate. Read the next part to fix it.


That said, I think it might be a good idea to do what the comment recommends. Allocate the memory in this function (rename the function to `createStartMenu` or something like that) and return the pointer to the Menu that was allocated. Don't forget to create a `destroyStartMenu` function that accompanies this.

Refer to this file for naming conventions: https://github.com/SuchithSridhar/Data-Structures-and-Algorithms/blob/master/C-ds/src/LinkedList/linkedList.h 
Note, I have not used javadoc style comments on those - you should be.

Reply:
I have fixed this issue and made these adjustments to the functions!